packages/runtime/runtime-utils/src/summaryUtils.ts

```diff
@@ -412,7 +412,9 @@ export function convertSummaryTreeToITree(summaryTree: ISummaryTree): ITree {
 			}
 
 			case SummaryType.Handle: {
-				throw new Error("Should not have Handle type in summary tree");
+				throw new Error(
```

Comment:
Is this an indicator of an internal bug could it be caused by a customer using something wrong? It should probably be either an assertion of a UsageError.

Reply:
Ah, I added this for debugging but forgot to remove it. Removed in now.